doc/components.md

```diff
@@ -208,3 +208,41 @@ The corners are drawn at the same size, the sides are stretched on the side dire
 Using this, you can get a box/rectangle that expands well to any sizes. This is useful for making panels, dialogs, borders.
 
 Check the example app `nine_tile_box` details on how to use it.
+
+## Effects
+
+Flame provides a set of effects that can be applied to a certain type of components, these effects can be used to animate some properties of your components, check below for a list of those effects.
+
+Examples of the running effects can be found [here](/doc/examples/effects);
+
+### MoveEffect
```

Comment:
I think we can leave this intro and create an effects.md file, as we probably gonna add several more effects soon.